src/views/Nft/market/Profile/ConnectedProfile.tsx

```diff
@@ -32,13 +29,11 @@ const TabMenuWrapper = styled(Box)`
 `
 
 const ConnectedProfile = () => {
-  const { profile, isLoading: isProfileLoading } = useProfile()
-  const { achievements, achievementFetchStatus } = useAchievements()
   const { account } = useWeb3React()
-  const { userNftsInitializationState, nfts: userNfts } = useUserNfts()
-
-  useFetchAchievements()
-  useFetchUserNfts()
+  const { isLoading: isProfileLoading, profile } = useProfile()
+  const { lastUpdated, setLastUpdated: refresh } = useLastUpdated()
+  const { achievements, isFetching: isAchievementsFetching } = useAchievementsForAddress(account, lastUpdated)
```

Comment:
can you take a look on https://github.com/pancakeswap/pancake-frontend/commit/05605c0da344cb6ebdfdab8514bb979e435a738a
you can get `mutate` as refresh in swr
```jsx
const { data, status, mutate } = useSWRImmutable([address, 'achievements'], () => getAchievements(address))
return {
 mutate: refresh
}
```

Reply:
Done